projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.ts

```diff
@@ -164,15 +151,16 @@ export class CommentComponent implements OnChanges, OnDestroy {
       };
       this.updated.emit(payload);
       this.hasUnsavedChanges = false;
-      this.editable = false;
+      this._editable = false;
```

Comment:
if there is no setter for these we can just make them selected and editable and get rid of the underscores

Reply:
ok